geoportailv3/lib/search.py

```diff
@@ -0,0 +1,79 @@
+from elasticsearch import Elasticsearch
+
+
+ES_ANALYSIS = {
+    'analysis': {
+        'filter': {
+            'ngram_filter': {
+                'type': 'nGram',
+                'min_gram': 2,
+                'max_gram': 20,
+                'token_chars': [
+                    'letter',
+                    'digit',
+                    'punctuation',
+                    'symbol'
+                ]
+            }
+        },
+        'analyzer': {
+            'ngram_analyzer': {
+                'type': 'custom',
+                'tokenizer': 'whitespace',
+                'filter': [
+                    'lowercase',
+                    'asciifolding',
+                    'ngram_filter'
+                ]
+            },
+            'whitespace_analyzer': {
+                'type': 'custom',
+                'tokenizer': 'whitespace',
+                'filter': [
+                    'lowercase',
+                    'asciifolding'
+                ]
+            }
+        }
+    }
+}
+
+ES_MAPPINGS = {
+    'poi': {
+        'properties': {
+            'object_id': {'type': 'string', 'index': 'not_analyzed'},
+            'layer_name': {'type': 'string', 'index': 'not_analyzed'},
+            'label': {
+                'type': 'string',
+                'index_analyzer': 'ngram_analyzer',
+                'search_analyzer': 'whitespace_analyzer'
+                },
+            'public': {'type': 'boolean', 'index': 'not_analyzed'},
+            'params': {'type': 'string', 'index': 'not_analyzed'},
+            'role_id': {'type': 'integer', 'index': 'not_analyzed'},
+            'ts': {'type': 'geo_shape'},
+        }
+    }
+}
+
+
+def get_es(request):
```

Comment:
I'd rather name this `get_eleasticsearch`, to make the code more explicit and readable.